License2Deploy/rolling_deploy.py

```diff
@@ -52,6 +52,7 @@ def __init__(self,
     self.health_wait = health_wait
     self.only_new_wait = only_new_wait
     self.existing_instance_ids = []
+    self.old_desired_capacity = 2
```

Comment:
The default can be anything because that field will be assigned to at the beginning of deployment.  However a test case is going to fail if set it to None.